tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -41,9 +45,12 @@ export class CardFobControllerComponent {
   @ViewChild('endFobWrapper') readonly endFobWrapper!: ElementRef;
   @Input() axisDirection!: AxisDirection;
   @Input() timeSelection!: TimeSelection;
-  @Input() cardAdapter!: CardFobAdapter;
+  @Input() cardFobHelper!: CardFobGetStepFromPositionHelper;
+  @Input() axisPositionFromStartStep!: number;
```

Comment:
Consider this wordy but more accurate name: timeSelectionStartStepAxisPosition.
Similarly: timeSelectionEndStepAxisPosition.

or combine them into a single object TimeSelectionAxisPositions: {
  start: number;
  end: number;
}

(I started wondering what the difference is supposed to be between lowestStep/startStep and highestStep/endStep, then realized startStep and endStep were specifically related to TimeSelection).